.github/workflows/validate.yaml

```diff
@@ -83,10 +83,7 @@ jobs:
         env:
           REVIEWDOG_GITHUB_API_TOKEN: ${{ github.token }}
         run: |
-          find . -type f \
-            -path './hack/*.*sh' \
-            -o -path './test/*.*.sh' \
-            -not -path '*vendor*' | xargs -r shellcheck --format=checkstyle \
+          find . -type f -path './**/*.*sh' -not -path '*vendor*' | xargs -r shellcheck --format=checkstyle \
```

Comment:
Why this reformatting here?

Reply:
easier to read (imo anyway)

Reply:
For me the opposite is true. The original is easier to read and understand. :rofl:

Reply:
Changed :)